gubernator/github/handlers.py

```diff
@@ -207,8 +215,13 @@ def emit_events(self, repo, number):
             action = body_json.get('action')
             sender = body_json.get('sender', {}).get('login')
             self.response.write('<tr><td>%s\n' % '<td>'.join(str(x) for x in
-                [event.timestamp, event.event, action, sender,
-                 '<pre>' + cgi.escape(body)]))
+                [   # Table columns
+                    event.timestamp,
+                    '%s<br><pre>%s</pre>' % (event.event, event.guid),
```

Comment:
nit: I'd use `<code>` instead of `<pre>` unless you think this is going to contain significant whitespace (which I'd really hope it does not, given it's supposed to be a guid).